pkg/acsengine/engine.go

```diff
@@ -283,6 +284,7 @@ func prepareTemplateFiles(properties *api.Properties) ([]string, string, error)
 	var files []string
 	var baseFile string
 	if properties.OrchestratorProfile.OrchestratorType == api.DCOS188 ||
+		properties.OrchestratorProfile.OrchestratorType == api.DCOS190 ||
```

Comment:
@wbuchwalter , can you also update "if strings.HasPrefix" condition (~line 359) for default "case api.DCOS" to be DCOS190 URL?